EasyFinance.Common.Tests/AccessControl/UserBuilder.cs

```diff
@@ -23,6 +23,12 @@ public UserBuilder AddLastName(string lastName)
             return this;
         }
 
+        public UserBuilder AddEmail(string email)
+        {
+            this.user.SetEmail(email);
```

Comment:
```suggestion
            this.user.Email = email;
```
I suggest to just set email directly because I think that is possible in this scenario. let me know if I'm wrong.